blog.html

```diff
@@ -0,0 +1,119 @@
+
+<!doctype html>
+<html lang="en"><head>
+  <meta charset="utf-8">
+  <title>Blog</title>
+  <link rel="stylesheet" href="styles/stylesheet.css">
+  <link href="https://fonts.googleapis.com/css2?family=Neuton:wght@200&display=swap" rel="stylesheet">
+</head>
+<body>
+  <div>
+    <header>
+      <img src="images/adoptadog-logo.svg" alt="Smiley face" height="35%" width="35%">
+      <img class = "imageclass"  src="images/cart-icon.svg" alt="Smiley face">  <span class = "headertextspan">$123.456</span>
+      <button class="button-checkout"> <a class = "abutton" href="checkout.html"> Checkout </a></button>
+
+      <p class="headerp"> Seattle's premier dog adoption agency </p>
+    </header>
+
+  <nav class="menu">
+    <ul class="navlist">
+      <li><a href="index.html">Home  </a></li>
+      <li><a href="dogs.html">Dogs</a></li>
+      <li><a href="blog.html">Blog</a></li>
+    </ul>
+  </nav>
+  <div>
+    <section class = "sectionblog">
+      <h1 class="dogsh1"> Adoptadog Blog </h1>
+      <div class="clearfix">
+      <img class="mainimage" src="images/blog-1.jpg" alt="Smiley face" height="35%" width="35%">
```

Comment:
You can assign % values in css, but in html theses should just be numbers and refer to px size.

Reply:
I would recommend not using height and width attributes and instead leave it to the html styling for sizing.